platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ConcurrentBuildsArtifactTransformIntegrationTest.groovy

```diff
@@ -147,28 +153,39 @@ block2.mustRunAfter blueThings
         result2.output.count("Transforming thing.jar to Blue") == 1
     }
 
+    /**
+     * With current cache implementation we don't use locking for cache entries (Gradle 8.6+)
```

Comment:
🤔 Given the comment, I assume we have tests for that elswhere, right?

If that's the case, then I would consider simply removing this integration test.
If it brings value, then we should full edit it so it works. Leaving a test in the `@ToBeImplemented` state for something that is already implemented is a sure way that it never gets fixed. And then back to the deletion suggestion ...

Reply:
I think there is a value in that, to test that concurrent builds don't fail when running the same artifact transform. This tests currently passes, since `@ToBeImplemented` doesn't change test result from "failure" to "success".

But I can remove it and change the test name

Reply:
WDYT about latest changes? I left a comment since I find it useful, but I can also delete that